test_accounts/appeals_apis/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,45 @@
+| Wesley     | Ford      | 1 Higher Level Review, 1 Supplemental Claim, 2 Legacy Appeals | 1012832025V743496 | va.api.user+idme.025@gmail.com | va.api.user+025@gmail.com | [46OPJSEX7URKZVICEKCARJVV7YRQRLFK](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B025%40gmail.com%29?secret=46OPJSEX7URKZVICEKCARJVV7YRQRLFK) |
+| Hector     | Allen     | 1 Higher Level Review, 1 Legacy Appeal                        | 1012667122V019349 | va.api.user+idme.011@gmail.com | va.api.user+011@gmail.com | [GFXLKZNSM3MLI35OP3XKVN4XEUKDIWI5](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B011%40gmail.com%29?secret=GFXLKZNSM3MLI35OP3XKVN4XEUKDIWI5) |
+| Janet      | Moore     | 1 Legacy Appeal                                               | 1012861229V078999 | va.api.user+idme.002@gmail.com | va.api.user+002@gmail.com | [B5C3L42PLUWO3U5T](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B002%40gmail.com%29?secret=B5C3L42PLUWO3U5T)                                 |
+| Tamara     | Ellis     | 1 Legacy Appeal                                               | 1012667145V762142 | va.api.user+idme.001@gmail.com | va.api.user+001@gmail.com | [LKI7FZ7ZEVRLGQRW](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B001%40gmail.com%29?secret=LKI7FZ7ZEVRLGQRW)                                 |
+| Jesse      | Gray      | 0 Appeals                                                     | 1012666073V986297 | va.api.user+idme.004@gmail.com | va.api.user+004@gmail.com | [7OMSKULT5PSVFE3SINTWBT2YA2MSFXU4](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B004%40gmail.com%29?secret=7OMSKULT5PSVFE3SINTWBT2YA2MSFXU4) |
+| Pauline    | Foster    | 404 Veteran Not Found                                         | 1012845630V900607 | va.api.user+idme.005@gmail.com | va.api.user+005@gmail.com | [RJIZ4BAI6OIYVKKGYYNHTR7F62IERCDT](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B005%40gmail.com%29?secret=RJIZ4BAI6OIYVKKGYYNHTR7F62IERCDT) |
+| Greg       | Anderson  | 500 Internal Server Error                                     | 1012666182V203559 | va.api.user+idme.008@gmail.com | va.api.user+008@gmail.com | [DU5XKE3SS47HHVF62X4C7FASIIAMOSVM](https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/Login.gov%20%28va.api.user%2B008%40gmail.com%29?secret=DU5XKE3SS47HHVF62X4C7FASIIAMOSVM) |
```

Comment:
Feedback on this table (even on the ordering of the columns) is welcome.

Reply:
This is a little difficult because Appeals Status v0 expects an `X-VA-SSN` header while v1 expects an `icn` query parameter - I know that we have no current consumers of Appeals Status v0, but Robin's team has been testing it in staging. If they're the only team that will use it, I think we could leave out an SSN column here, but if there's a chance that any other team could onboard with v0 instead of v1, I do think an SSN column wouldn't hurt.

Reply:
This page is intended for Appeals Status API v1 only, so ICN should be enough. The Appeals Status v0 docs would continue to link to the [general Benefits and Appeals Test Account page](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts/benefits_test_accounts.md).

Reply:
If you were testing something, is there a column you'd want to see first or is it contextual to what you're doing?  That said, there aren't so many columns that it seems burdensome as is.

Reply:
@melliroo Casey and I reviewed the column order while considering that question yesterday and landed on the following order:
- Sandbox Response
- ICN
- ID.me Username
- Login.gov Username
- First Name
- Last Name

I've made those changes in the document.